README.md

```diff
@@ -1 +1,37 @@
-# spec
+# OpenEvents
+
+Events are everywhere.  However, event publishers tend to describe events differently.
+
+The lack of a common way of describing events means developers must constantly re-learn how to receive events.  This also limits the potential for libraries, tooling and infrastructure to aide the delivery of event data across environments, like SDKs, event routers or tracing systems.  The portability and productivity we can achieve from event data is hindered overall.
```

Comment:
Can we wrap things at 80 columns? it makes the github diff tool work much better.

Reply:
Yes!